packages/sdk-snippets/tsconfig.build.json

```
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "moduleResolution": "nodenext",
```

typescript will be changing the behavior of `NodeNext` in upcoming versions so i'd recommend pinning it like so:

```suggestion
    "moduleResolution": "Node16",
```